NameLayerMain/src/vg/civcraft/mc/namelayer/GroupManager.java

```diff
@@ -121,64 +135,70 @@ public void run() {
 		}
 	}
 	
+	public static List<Group> getSubGroups(String name) {
+		List<Group> groups = groupManagerDao.getSubGroups(name);
+		for (Group group : groups) {
+			groupsByName.put(group.getName().toLowerCase(), group);
+			groupsById.put(group.getGroupId(), group);
+		}
+		return groups;
+	}
+	
 	/*
 	 * Making this static so I can use it in other places without needing the GroupManager Object.
 	 * Saves me code so I can always grab a group if it is already loaded while not needing to check db.
 	 */
-	public static Group getGroup(String groupName){
-		groupName = groupName.toLowerCase();
-		if (groupsByName.containsKey(groupName))
-			return groupsByName.get(groupName);
-		else{ 
-			Group group;
-			group = groupManagerDao.getGroup(groupName);
+	public static Group getGroup(String name){
+		String lower = name.toLowerCase();
+		if (groupsByName.containsKey(lower)) {
+			NameLayerPlugin.log(Level.INFO, "Group " + name + " found");
```

Comment:
Yeah, that was for debugging purposes, I added them in several places and forgot to take them out.